### ISSUE TITLE
Samples filtered down to 0 sequences in otu_table_w_tax_no_pynast_failures.biom

### ISSUE THREAD

**Issue by lkursell** [biocore/qiime#2161]
An unusual behavior from the default `pick_open_reference_otus.py` pipeline. Samples which are seen with low counts after `min_count` filtering can then be preserved in the final biom table even though they have no sequences. As an example:

```
biom summarize-table -i otu_table_mc2.biom | grep "sampleX"
sampleX: 2.0

biom summarize-table -i otu_table_mc2_w_tax_no_pynast_failures.biom | grep "sampleX"
sampleX: 0.0
```

This initially lead me to some confusion about why some samples with no sequences are preserved in the biom table, but other samples, like those with 0 sequences from `split_libraries.py` are not carried through.

**Comment by antgonza** [comment]
Yes, I have encountered this too. Not sure if a bug or desirable and if qiime or biom but agree that is annoying and leads to confusion. 👍

**Comment by gregcaporaso** [comment]
I think it's just something that slipped through - I wouldn't really call it a bug, since it's correct, it's just strange since it's different than how this is usually handled in QIIME. I think we should stick with filtering (``filter_samples_from_otu_table.py -n 1`` would let you work around this) for now, since the focus is on QIIME 2 development now except for critical bug fixes.
